DataConnectors/esetSmc.json

```diff
@@ -0,0 +1,117 @@
+{
+    "id": "EsetSMC",
+    "title": "Eset Security Management Center",
+    "publisher": "Eset",
+    "descriptionMarkdown": "Connector to bring Eset SMC data into Sentnel using custom table for easy quering. Data is sent from Eset SMB in JSON format to syslog connector VM and FluentD parsing is used so data are sent structured to Sentinel.",
+    "graphQueries": [
+        {
```

Comment:
Looks like the data lands in custom logs. Please update this to insert in the DATATYPE_NAME which is eset_CL for both legend and baseQuery.